tests/base.py

```diff
@@ -1,24 +1,24 @@
-import tap_tester.connections as connections
-import tap_tester.menagerie as menagerie
-import tap_tester.runner as runner
+from tap_tester import connections, menagerie, runner
 import os
 import unittest
 from datetime import datetime as dt
 import time
 
 class AsanaBase(unittest.TestCase):
+    """
+    Base class for tap.
+    """
+
     START_DATE = ""
-    DATETIME_FMT = {
-        "%Y-%m-%dT%H:%M:%SZ",
-        "%Y-%m-%d %H:%M:%S",
-        "%Y-%m-%dT%H:%M:%S.%fZ"
-    }
     PRIMARY_KEYS = "table-key-properties"
     REPLICATION_METHOD = "forced-replication-method"
     REPLICATION_KEYS = "valid-replication-keys"
     FULL_TABLE = "FULL_TABLE"
     INCREMENTAL = "INCREMENTAL"
     OBEYS_START_DATE = "obey-start-date"
+    START_DATE_FORMAT = "%Y-%m-%dT00:00:00Z"
+    REPLICATION_DATE_FOMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+    BOOKMARK_FOMAT = "%Y-%m-%dT%H:%M:%S.%f"
 
     first_start_date = '2019-01-01T00:00:00Z'
     second_start_date = '2020-08-15T00:00:00Z'
```

Comment:
Instead of static values of start dates, can't we just make them dynamic.
Something like this :
first_start_date = start_date = datetime.strftime(utils.now() - timedelta(days=365), START_DATE_FORMAT)
So that integration tests will not take longer by each passing day.

Reply:
Test data used for this test is static so we don't need to configure start_date dynamically.